src/test/scala/uk/gov/hmrc/test/ui/pages/returns/ReturnsPage.scala

```diff
@@ -171,4 +171,21 @@ object ReturnsPage extends BasePage {
     SharedActions.clickButton()
     this
   }
+
+
+  def assertEclReturnNumber(value: String): this.type = {
+    val actual = getText(
+      By.xpath(s"//p[contains(text(),'ECL return number: ')]")
+    )
+    assert(actual.contains(value))
+    this
+  }
+
+  def assertAmountNeedToPay(value: String): this.type = {
+    val actual = getText(
+      By.xpath(s"//p[contains(text(),'Amount you need to pay: £10,000')]")
```

Comment:
Why is the amount fixed at £10,000? Is this the only liability scenario tested? Would it not be more flexible to simply match on the text excluding the amount (like is done for the ECL return number in the above check)?

Reply:
done